source/manual/alerts/email-alerts.html.md

```diff
@@ -8,8 +8,7 @@ last_reviewed_on: 2018-08-31
 review_in: 6 months
```

Comment:
Worth updating the last reviewed on date as well I think.

Reply:
fixed in 31e375d